project/plugins.sbt

```diff
@@ -1,5 +1,4 @@
 addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.0.0-RC1")
-addSbtPlugin("com.typesafe.sbt" % "sbt-bintray-bundle" % "1.0.2")
 addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "3.0.0")
 addSbtPlugin("com.github.ddispaltro" % "sbt-reactjs" % "0.5.2")
 addSbtPlugin("com.lightbend.conductr" % "sbt-conductr" % "2.1.4")
```

Comment:
It's 2.1.7 in [this PR](https://github.com/lagom/activator-lagom-java-chirper/pull/20/files#diff-a84f91f20f040218bccd09fed4761fb3R5) ;-)